Sources/BrowserServicesKit/Domain/AppVersion.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  AppVersion.swift
+//  DuckDuckGo
+//
+//  Copyright © 2017 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+/// 1. App version is moved to BSK and it's assumed Mac can use this struct without any changes.
+/// 2. Create AppVersion protocol in BSK and create struct that implements it in Core app called iOSAppVersion.swift. (APMobile)
```

Comment:
I'm not sure I understand this comment

Reply:
This was an earlier comment I forgot to remove, sorry for the confusion